examples/chart/teleport-kube-agent/Chart.yaml

```diff
@@ -1,7 +1,9 @@
+.version: &version "11.0.0-dev"
+
 name: teleport-kube-agent
 apiVersion: v2
-version: "11.0.0-dev"
-appVersion: "11.0.0-dev"
+version: *version
```

Comment:
I swicthed to anchors for the `teleport-cluster` chart and its subchart dependency.

```yaml
version: v10.0.0-dev
appVersion: v10.0.0-dev
# ....
dependencies:
  - name: teleport-operator
    version: v10.0.0-dev
    condition: installCRDs
```

It was not really safe to sed without taking care of the indentation, it would have caught any other dependency. Hence I added an anchor and modifierd the `version.mk` logic to edit the anchor instead of the values themselves. While this si not needed for `teleport-kube-agent` I think it's in our best interest to set version on all charts using the same mechanism.